src/elements/developer-instructions.md

```diff
@@ -0,0 +1,148 @@
+# Developer instructions
+
+## Development mode installation
+
+- This method allows you to modify the source code for example DataJoint
+  workflows (e.g. `workflow-array-ephys`, `workflow-calcium-imaging`) and their
+  dependencies (i.e., DataJoint Elements).
+
+- Launch a new terminal and change directory to where you want to clone the
+  repositories
+  `bash cd ~/Projects `
+
+- `workflow-array-ephys`
+    <details>
+    <summary>Click to expand details</summary>
+
+  - Clone the repositories
+
+    ```bash
+    git clone https://github.com/datajoint/element-lab
+    git clone https://github.com/datajoint/element-animal
+    git clone https://github.com/datajoint/element-session
+    git clone https://github.com/datajoint/element-interface
+    git clone https://github.com/datajoint/element-array-ephys
+    git clone https://github.com/datajoint/workflow-array-ephys
+    ```
+
+  - Install each package with the `-e` option
+
+    ```bash
+    pip install -e ./element-lab
+    pip install -e ./element-animal
+    pip install -e ./element-session
+    pip install -e ./element-interface
+    pip install -e ./element-array-ephys
+    pip install -e ./workflow-array-ephys
+    ```
+
+    </details>
+
+- `workflow-calcium-imaging`
+    <details>
+    <summary>Click to expand details</summary>
+
+  - Clone the repositories
+
+    ```bash
+    git clone https://github.com/datajoint/element-lab
+    git clone https://github.com/datajoint/element-animal
+    git clone https://github.com/datajoint/element-session
+    git clone https://github.com/datajoint/element-interface
+    git clone https://github.com/datajoint/element-calcium-imaging
+    git clone https://github.com/datajoint/workflow-calcium-imaging
+    ```
+
+  - Install each package with the `-e` option
+
+    ```bash
+    pip install -e ./element-lab
+    pip install -e ./element-animal
+    pip install -e ./element-session
+    pip install -e ./element-interface
+    pip install -e ./element-calcium-imaging
+    pip install -e ./workflow-calcium-imaging
+    ```
+
+    </details>
```

Comment:
maybe this is overcomplicating the code, but I think we can cut this down for developers

```suggestion
- Clone the relevant workflow and refer to the `requirements.txt` in the workflow for the list of Elements to clone and install as editable. You will also need to install `element-interface` 
    ```bash
    deps=("lab" "animal" "session" "interface" "<others>")
    for repo in $deps # clone each
    do 
        git clone https://github.com/datajoint/element-$repo
    done
    for repo in $(ls -d ./{element,workflow}*) # editable install 
    do 
        pip install -e ./$repo
    done
    ```
```